src/pages/workspaces/workspace/Analyses.js

```diff
@@ -505,14 +507,14 @@ const Analyses = _.flow(
           }
         }),
         exportingAnalysisName && h(ExportAnalysisModal, {
-          printName: getDisplayName(exportingAnalysisName),
+          printName: getFileName(exportingAnalysisName),
```

Comment:
`AnalysisDuplicator` uses `renamingAnalysisName` or `copyingAnalysisName` directly for its `printName` prop.  `ExportAnalysisModal` takes `getFileName(exportingAnalysisName)`. Previously, all 3 cases were `getDisplayName(stateVariable)`. Why the inconsistency now?